dulwich/objects.py

```diff
@@ -1510,6 +1510,18 @@ def sign(self, keyid: Optional[str] = None) -> None:
                     self.as_raw_string(), mode=gpg.constants.sig.mode.DETACH
                 )
 
+    def without_sig(self) -> "Commit":
+        """Return a Commit object copy with the signature (GPG or SSH) removed.
+
+        self.gpgsig is a signature for the raw string serialization bytes of the
```

Comment:
this sentence doesn't make sense to me

Reply:
One challenge I have is there are multiple different computing concepts that use the words "message" or "signature". In an earlier version of this text I used the phrase "cryptographic message" to refer to the "message" from the perspective of cryptographic signing. Do you find the phrase "cryptographic message" confusing? I worry it will be confused with the git commit message.

Reply:
Actually, nevermind. I think I understand what is trying to say upon rereading it.

Reply:
Here's a more wordy alternative:

"The raw byte string serialization of the returned Commit object is the cryptographic message for which self.gpgsig is the signature."

Do you think this alternative is more understandable?

Reply:
This forced-pushed squashed [934b017](https://github.com/jelmer/dulwich/commit/934b017555e168eeff548af33d323aeeac5e2e8e) of this PR branch has a very tiny wording tweak.

The alternative wording above is in a further commit as branch https://github.com/castedo/dulwich/tree/commit_without_sig2 .

I can't tell which wording is better because I'm not sure how well "cryptographic message" is understood as intended.